src/components/vm/consoles/vnc.tsx

```diff
@@ -315,7 +369,11 @@ const VncFooter = ({ vm, vnc, inactive_vnc, onAddErrorNotification }) => {
                         vm={vm}
                         console={vnc}
                         url={vnc && cockpit.format("vnc://$0:$1", connection_address(), vnc.port)}
-                        onEdit={() => Dialogs.show(<VncEditModal vm={vm} inactive_vnc={inactive_vnc} />)}
+                        onEdit={
+                            inactive_vnc
+                                ? () => Dialogs.show(<VncEditModal vm={vm} inactive_vnc={inactive_vnc} />)
+                                : null
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2267-42a6eadf-20250808-101729-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)